boa/src/syntax/ast/keyword.rs

```diff
@@ -433,6 +434,21 @@ pub enum Keyword {
     Yield,
 }
 
+impl Keyword {
+    pub fn as_binop(self) -> Option<BinOp> {
+        match self {
+            Keyword::In => Some(BinOp::Comp(CompOp::In)),
+            _ => None,
+        }
+    }
+}
+
+impl Into<BinOp> for Keyword {
```

Comment:
An `Into` trait must not fail, I think that with the `as_binop()` method is enough.